src/ast/location.h

```diff
@@ -84,4 +63,49 @@ class Location {
   std::shared_ptr<ASTSource> source_;
 };
 
+class LocationChain;
+using Location = std::shared_ptr<LocationChain>;
+
+// LocationChain is a list of locations.
```

Comment:
You know I'm gonna comment on this. It is more of a LINKED list than a "list". Chain describes it well enough for me so this comment should go byebye.

Reply:
A linked list is a list :)I don’t feel strongly, I can drop the comment.

Reply:
drop it like it's hot!

Reply:
I wound up updating the comment to make it useful.

Reply:
Much better. Thank you.